### ISSUE TITLE
Email extracter

### ISSUE THREAD

**Issue by kolatubosun** [Yorubaname/yorubaname-dashboard#18]
I need a way to download, at once, all the emails of the people who have added names to the dictionary, so I can subscribe them to our newsletter.
![screenshot 115](https://cloud.githubusercontent.com/assets/18510601/17363810/9860939a-5974-11e6-80c9-e7604c3c02ac.png)

Just a button to download it as excel would do.

**Comment by dadepo** [comment]
I do not think it is a good practice to add someone to a newsletter because that person submitted a name (and in doing so gave us their email address). 

The feature can still be implemented but I doubt if it should be used for the use case described.

**Comment by kolatubosun** [comment]
Okay. Fair point. I do hope though, as described in [this website issue](https://github.com/Yorubaname/yorubaname-website/issues/43) that we can create small print notifications that by submitting their emails they will be subscribed to our newsletter. Or create checkboxes that can allow them opt out before they click submit.

What I've realised from trying to put some of these emails into the newsletter is that most of them (certainly all on that page I captured) already signed up themselves. So, in some way, we're providing a form of service by letting them kill two birds with one stone.
